CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Support for OpenShift 4.3 and 4.5.
   [conjurdemos/kubernetes-conjur-demo#122](https://github.com/conjurdemos/kubernetes-conjur-demo/issues/122)
+- Support for `SQLMode = prefer` in PostgreSQL
```

Comment:
Let's update this to be under `### Fixed` since this is a bug. I think we should be more descriptive and provide some detail about the fix. Like how it fixes the error reporting for when PostgreSQL clients attempt to connect to Secretless with TLS, and now Secretless correctly informs the client that TLS is not supported.

Reply:
Please link to the issue in the CHANGELOG entry, like the one above.

Reply:
fixed